saleor/payment/tasks.py

```diff
@@ -0,0 +1,22 @@
+from celery.utils.log import get_task_logger
+
+from .utils import release_payment, get_unfinished_payments
+from ..celeryconf import app
+from .models import Payment
+
+
+task_logger = get_task_logger(__name__)
+
+@app.task
+def release_unfinished_payments_task():
+    payments = get_unfinished_payments()
+    for payment in payments.iterator():
+        release_dangling_unfinished_payment_task.apply(payment.pk)
+
+
+@app.task
+def release_dangling_unfinished_payment_task(pk):
+    payment = Payment.objects.get(pk=pk)
+
+    release_payment(payment)
```

Comment:
This function throws exceptions, how do we want to handle them?

Reply:
Yes, fixed